src/main/kotlin/com/hibob/academy/employeeFeedback/dao/models.kt

```diff
@@ -4,6 +4,14 @@ import java.sql.Date
 
 data class EmployeeInfo(val id: Int, val role: RoleType, val department: String, val companyId: Int)
 
+data class EmployeeCreation(val firstName: String, val lastName: String, val role: RoleType, val department: String, val companyId: Int)
```

Comment:
It's better to call it EmployeeCreationRequest, so it's clear that it holds data, not doing an action